index.js

```diff
@@ -2,7 +2,7 @@ import escapeStringRegexp from 'escape-string-regexp';
 import getHomeDirectory from '#home-directory';
 
 const extractPathRegex = /\s+at.*[(\s](.*)\)?/;
-const pathRegex = /^(?:(?:(?:node|node:[\w/]+|(?:(?:node:)?internal\/[\w/]*|.*node_modules\/(?:babel-polyfill|pirates)\/.*)?\w+)(?:\.js)?:\d+:\d+)|native)/;
+const pathRegex = /^(?:(?:node|node:[\w\/]+|(?:(?:node:)?internal\/[\w\/]*|ext:[\w\/]+|https:\/\/jsr\.io\/[\w\/@.]+|.*node_modules\/(?:babel-polyfill|pirates)\/.*)?\w+)(?:\.[jt]s)?:\d+:\d+|native)/;
```

Comment:
This is more readable:

```suggestion
const pathRegex = /^(?:(?:node|node:[\w\/]+|(?:(?:node:)?internal\/[\w\/]*|ext:[\w\/]+|https:\/\/jsr\.io\/[\w\/@.]+|.*node_modules\/(?:babel-polyfill|pirates)\/.*)?\w+)(?:\.(?:js|ts))?:\d+:\d+|native)/;
```